docs/security/cve/zipbombs-and-octopus-deploy.md

```diff
@@ -0,0 +1,59 @@
+---
+title: ZipBombs and Octopus Deploy
+description: Octopus Server actively prevents ZipBomb-based denial-of-service attacks using a number of preventative measures.
+position: 1
+---
+
+We take every reasonable effort to make Octopus Deploy secure against well-known vulnerabilities. One such vulnerability is denial-of-service attacks via ZipBombs.
+
+## What is a ZipBomb?
+
+Using a specially crafted archive file (eg .zip, .jar, .tar.gz), a malicious actor could potentially cause a machine to become unresponsive by consuming large amounts of CPU and disk space. For more information about Zip Bombs, refer to the following:
+
+- [ZipBombs according to OWASP](https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/10-Business_Logic_Testing/09-Test_Upload_of_Malicious_Files#zip-bombs)
+- [ZipBombs according to Security Researcher, David Fifield](https://www.bamsoftware.com/hacks/zipbomb/)
+
+## Does Octopus Deploy prevent ZipBomb attacks?
+
+Yes. The Octopus Server, and deployment agents that run on Workers and deployment targets, are designed to detect and prevent extraction of potentially malicious archives.
```

Comment:
I noticed `Workers` got a capital, as it's an Octopus concept. But `deployment target` did not. This is very minor, but I wondered if these two should be consistent?